src/Slider.jsx

```diff
@@ -464,14 +465,14 @@ class Rheostat extends React.Component {
     this.setStartSlide(ev, ev.clientX, ev.clientY);
 
     if (typeof document.addEventListener === 'function') {
-      document.addEventListener('mousemove', this.handleMouseSlide, false);
```

Comment:
what happens with this change in browsers that don't support passive event listeners?